packages/create-project/src/create-project.ts

```diff
@@ -184,43 +192,26 @@ export async function createApp({
 
   if (await tryGitInit(root, gitInit)) {
     console.log('Initialized a git repository.');
-    console.log();
   }
 
-  let cdpath: string;
-  if (path.join(originalDirectory, appName) === appPath) {
-    cdpath = appName;
-  } else {
-    cdpath = appPath;
-  }
+  const messageFile = path.join(root, '.post-create');
 
-  console.log(`${chalk.green('Success!')} Created ${chalk.bold(appName)} at ${chalk.bold(appPath + '/')}`);
-  console.log();
-  console.log('Inside that directory, you can run several commands:');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}build`));
-  console.log('    Builds all the code.');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} start`));
-  console.log('    Runs the built Worker.');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}workflow`));
-  console.log('    Starts a Workflow.');
-  console.log();
-  console.log('To begin development, start Temporal Server:');
   console.log();
-  console.log(chalk.cyan('  cd'), '~/path/to/temporal/docker-compose/');
-  console.log(`  ${chalk.cyan('docker-compose up')}`);
+  console.log(`${chalk.green('Success!')} Created project ${chalk.bold(appName)} at:`);
   console.log();
-  console.log(
-    chalk.dim.italic(
-      `If you haven't run Temporal Server before, visit:\nhttps://docs.temporal.io/docs/typescript/getting-started/`
-    )
-  );
-  console.log();
-  console.log(`Then, in the ${chalk.bold(cdpath + '/')} directory, using two other shells, run these commands:`);
-  console.log();
-  console.log(`  ${chalk.cyan(`${displayedCommand} ${useYarn ? '' : 'run '}start.watch`)}`);
-  console.log(`  ${chalk.cyan(`${displayedCommand} ${useYarn ? '' : 'run '}workflow`)}`);
+  console.log(chalk.bold(appPath + '/'));
   console.log();
+
+  try {
+    await access(messageFile);
+    const message = await readFile(messageFile);
```

Comment:
I'm definitely not blocking on this, but you might as well read the file as 'utf8' and get a string back.